Twig/Extension/SonataAdminExtension.php

```diff
@@ -437,6 +437,11 @@ protected function getTemplate(
         try {
             $template = $environment->loadTemplate($templateName);
         } catch (\Twig_Error_Loader $e) {
+            @trigger_error(
+                'Relying on default template loading on field template '.
+                'loading exception is deprecated and will be removed in 4.0',
```

Comment:
Oh ok, so forgot for the commented code.

But specify that a `\Twig_Error_Loader` exception will be thrown on 4.0 would be great.